### PR TITLE
Adapt to fix in `BindingStep`

### PR DESCRIPTION
**Pull request by jglick**
Prevents https://github.com/jenkinsci/credentials-binding-plugin/pull/143 from breaking a few tests. Noticed in https://github.com/jenkinsci/bom/pull/573.

### CODE REVIEW

**Review by dwnusbaum — APPROVED** [review]

**Comment by jglick** [comment]
Maintainers, please release when possible, so that this can get into the BOM and unblock `credentials-binding` updates.

This plugin seems like a good candidate for JEP-229…

**Review by car-roll — APPROVED** [review]

**Pull request merged**
